scheduler/core/components/ranker/default.py

```diff
@@ -81,6 +81,37 @@ def __post_init__(self):
             if self.altitude_limits[site][MinMax.MAX] > Angle(90.0*u.deg):
                 raise ValueError(f'The maximum altitude limit for {site.name} must be 90 degrees or less.')
 
+    def altitude_limits_to_str(self) -> str:
```

Comment:
Lets move this to `scheduler/core/output`

Reply:
Sure, I wasn't aware of the use of `scheduler/core/output`

Reply:
If I move this one to `scheduler/core/output`, I got an `ImportError`. There are circular imports.

In the Ranker default I am importing output -> output imports Collector -> Collector imports NightEvents -> NightEvents imports selection.py -> selection.py imports Ranker

Reply:
The import  can be done inside the __str__(self) method, but it won't be done yet, since NightEvent should be checked and maybe cleaned in the future

Reply:
Modified method to be private